### PR TITLE
Implement the "Preferences" tab on new user settings

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/riot-web/issues/8202
Part of https://github.com/vector-im/riot-web/issues/7605
~~**Built on https://github.com/matrix-org/matrix-react-sdk/pull/2494**~~
~~Relevant changes are https://github.com/matrix-org/matrix-react-sdk/compare/travis/usettings/tab/notifications...travis/usettings/tab/preferences~~

*Surprise! It's just a bunch of checkboxes and a lonely text field*

image

### CODE REVIEW

**Comment by Half-Shot** [comment]
That's a little intimidating (if slightly better than what we have now). Would it look any better with slightly more margin above the subtitles?

**Comment by turt2live** [comment]
Possibly. First pass is to sort these things then iterate upon it later. I'll add a note to the general "improve all this stuff" list.

**Comment by turt2live** [comment]
I'm actually going to pull this from the review queue until the prior PR lands just to keep things sane on the Git side of things.

File: src/components/views/settings/tabs/PreferencesSettingsTab.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    <span className="mx_SettingsTab_subheading">{_t("Room list")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.ROOM_LIST_SETTINGS)}
+
+                    <span className="mx_SettingsTab_subheading">{_t("Timeline")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.TIMELINE_SETTINGS)}
+
+                    <span className="mx_SettingsTab_subheading">{_t("Advanced")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.ADVANCED_SETTINGS)}
+                    {autoLaunchOption}
+                    <Field id={"autocompleteDelay"} label={_t('Autocomplete delay (ms)')} type='number'
```
Comment (jryans) [inline src/components/views/settings/tabs/PreferencesSettingsTab.js]: Is that Nunito or some other font for the value inside the autocomplete input? (I believe the design wants Nunito.)

2019-01-24 at 11 53

Comment (turt2live) [reply src/components/views/settings/tabs/PreferencesSettingsTab.js]: Looks like it is indeed using Arial for some reason. 

/me shall fix

Comment (turt2live) [reply src/components/views/settings/tabs/PreferencesSettingsTab.js]: Fixed 
image

**Pull request merged**